common/theme.js

```diff
@@ -0,0 +1,46 @@
+const themeKey = 'theme';
+const darkValue = 'dark';
+const whileValue = 'white';
```

Comment:
Is this supposed to be whiteValue? If so, it should probably be lightValue and light